pg_query.h

```diff
@@ -96,6 +96,7 @@ extern "C" {
 #endif
 
 PgQueryNormalizeResult pg_query_normalize(const char* input);
+PgQueryNormalizeResult pg_query_normalize_utility(const char* input);
```

Comment:
If there's potential for more variants in the future, we should follow the `pg_query_parse_opts` pattern and use an enum. But unless we want to offer fine-grained normalization (e.g. normalizing specific statement types like `SelectStmt`), this probably isn't needed.